src/services/resources-getter.js

```diff
@@ -34,11 +34,22 @@ function ResourcesGetter(model, options, params) {
       associations.push(associationFromSorting);
     }
 
+    const associationFields = flatten(Object.keys(model.associations)
```

Comment:
It looks like a duplicate with the HasManyGetter. This need to be refacto.
Feel free to use composition ahah 😜

Reply:
I cannot use composition without breaking the interface.

I'll make the change